tests/vkray/raygen-trace-ray-param-non-struct.slang

```diff
@@ -0,0 +1,96 @@
+//TEST:SIMPLE(filecheck=CHECK): -target hlsl -stage raygeneration -entry main
+
+RaytracingAccelerationStructure as;
+
+RWStructuredBuffer<float> outputBuffer1;
```

Comment:
Don't you need to initialize this variable with zeros?

Reply:
This test does not run any computation, we only check to ensure we properly emit a struct before and copy after a use-case.